api/src/middlewares/loggingMiddleware.js

```diff
@@ -1,24 +1,38 @@
+const logfmt = require('logfmt');
+
 const loggingMiddleware = async (req, res, next) => {
   const startTime = new Date();
-  let userlog = "";
   res.on("finish", async () => {
     try {
       const responseTimeMs = new Date() - startTime;
       req.responseTimeMs = responseTimeMs;
 
       if (req.body?.password) req.body.password = "**********";
 
-      if (req.user) {
-        const userRole = req.user.patches.modelName === "ReferentPatches" && "referent";
-        userlog = userRole ? ` UserRole: ${req.user?.role} | UserId: ${req.user.id} |` : ` UserId: ${req.user.id} |`;
-      }
-
       const ip = req.ipInfo;
 
+      let log = {
+        method: req.method,
+        url: req.originalUrl,
+        status: res.statusCode,
+        responseTime: `${req.responseTimeMs}ms`,
+        ip: ip
+      }
+
       const hasPayload = req.body && Object.keys(req.body).length > 0;
-      const payloadLog = hasPayload ? ` Payload: ${JSON.stringify(req.body)} |` : "";
+      if (hasPayload) {
+        log["payload"] = JSON.stringify(req.body)
```

Comment:
```suggestion
        log.payload = JSON.stringify(req.body)
```

P.S. : j'ai candidaté pour cette [mission-ci](https://www.welcometothejungle.com/fr/companies/communaute-beta-gouv/jobs/developpeur-euse-fullstack-experimente-e-pour-la-plateforme-snu_paris?q=36c821504a0b4c93ceecf9875b078e55&o=49345c87-0314-4aec-87cf-a37f06e0d7d3) via ce [formulaire-ci](https://tally.so/r/nrD112), et j'attends toujours une date d'entretien 😉